### PR TITLE
Add script of push cnab bunlde API test

### PR DESCRIPTION
**Pull request by danfengliu**
Signed-off-by: danfengliu <danfengl@vmware.com>
Fixed issue: #10762,#10763.

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -133,6 +133,10 @@ jobs:
           wget https://get.helm.sh/helm-v3.1.1-linux-386.tar.gz && tar zxvf helm-v3.1.1-linux-386.tar.gz && \
           sudo mv linux-386/helm /usr/local/bin/helm3 && \
           helm3 plugin install https://github.com/chartmuseum/helm-push
+          git clone https://github.com/cnabio/cnab-to-oci.git
```
Comment (ywk253100) [inline .github/workflows/CI.yml]: Could we use a fixed release version, in case the master branch may break our testing

Comment (danfengliu) [reply .github/workflows/CI.yml]: Yes, it's pinned to the latest version v0.3.0-beta4 now.

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -109,7 +109,9 @@ jobs:
           cd src/github.com/goharbor/harbor
           pwd
           go env
-          echo "::set-env name=GOPATH::$(go env GOPATH):$GITHUB_WORKSPACE"
+          echo "::set-env name=RUNNER_GO_PATH::$(go env GOPATH)/src/github.com/docker"
```
Comment (bitsf) [inline .github/workflows/CI.yml]: 这个RUNNER_GO_PATH 没有用

File: .github/workflows/CI.yml [hunk 3]
```diff
@@ -130,9 +132,14 @@ jobs:
           sudo cp ./tests/harbor_ca.crt /usr/local/share/ca-certificates/
           sudo update-ca-certificates
           sudo service docker restart
-          wget https://get.helm.sh/helm-v3.1.1-linux-386.tar.gz && tar zxvf helm-v3.1.1-linux-386.tar.gz && \
-          sudo mv linux-386/helm /usr/local/bin/helm3 && \
+          wget https://get.helm.sh/helm-v3.1.1-linux-386.tar.gz && tar zxvf helm-v3.1.1-linux-386.tar.gz
+          sudo mv linux-386/helm /usr/local/bin/helm3
           helm3 plugin install https://github.com/chartmuseum/helm-push
+          cd $RUNNER_GO_PATH && sudo git clone https://github.com/cnabio/cnab-to-oci.git
```
Comment (bitsf) [inline .github/workflows/CI.yml]: 不要用 RUNNER_GO_PATH

File: .github/workflows/CI.yml [hunk 4]
```diff
@@ -130,9 +132,14 @@ jobs:
...
+          cd cnab-to-oci && sudo git checkout v0.3.0-beta4
+          go list
+          sudo make build
```
Comment (bitsf) [inline .github/workflows/CI.yml]: 不要sudo

File: .github/workflows/CI.yml (hunk 3 above)
Comment (bitsf) [reply .github/workflows/CI.yml]: 不用sudo

File: .github/workflows/CI.yml (hunk 2 above)
Comment (danfengliu) [reply .github/workflows/CI.yml]: cd $RUNNER_GO_PATH 后面用到了

**Pull request merged**